plugin/src/withCoreAndroid.ts

```diff
@@ -26,21 +34,23 @@ const androidSdkImportMap: Record<string, string> = {
   '@adobe/react-native-aepuserprofile': 'import com.adobe.marketing.mobile.UserProfile',
   '@adobe/react-native-aepedge': 'import com.adobe.marketing.mobile.Edge',
   '@adobe/react-native-aepassurance': 'import com.adobe.marketing.mobile.Assurance',
-  '@adobe/react-native-aepedgeidentity': 'import com.adobe.marketing.mobile.EdgeIdentity',
-  '@adobe/react-native-aepedgeconsent': 'import com.adobe.marketing.mobile.EdgeConsent',
+  // todo to ask for its import as it is not in documentation
+  //'@adobe/react-native-aepedgeidentity': 'import com.adobe.marketing.mobile.EdgeIdentity',
```

Comment:
yes this will be needed, code: https://github.com/adobe/aepsdk-edgeidentity-android/blob/main/Documentation/getting-started.md

Reply:
added